server/api/products.js

```diff
@@ -6,7 +6,10 @@ const {
 // GET /api/products
 router.get('/', async (req, res, next) => {
   try {
-    const allProducts = await Product.findAll();
+    const allProducts = await Product.findAll({
+      Attributes: ['title'],
+      distinct: true,
```

Comment:
Is there more we need to do in order to get the price range?

Reply:
I just realized the seed file doesnt alter the prices based on the format, could we apply some front end logic to save from having to edit everything in the seed all over again or is that not negotiable

Reply:
I think I see what you're saying.  Could we talk about this more as a group on Monday?